aiida/backends/tests/test_nodes.py

```diff
@@ -1593,16 +1585,8 @@ def _check_existence(self, uuids_check_existence, uuids_check_deleted):
                 # I check that it raises
                 with self.assertRaises(NotExistent):
                     orm.load_node(uuid)
-        except Exception as exc:  # pylint: disable=broad-except
-            import sys
-            six.reraise(
-                type(exc),
-                exc.__class__((
-                    str(exc) + '\nCurrent UUID being processed: {}' + '\nFull uuids_check_existence: {}' +
-                    '\nFull uuids_check_deleted: {}'
-                ).format(uuid, uuids_check_existence, uuids_check_deleted)),
-                sys.exc_info()[2]
-            )
+        except Exception:  # pylint: disable=broad-except
```

Comment:
This try/except doesn't do anything, right?

Reply:
Correct, removed